packages/lib/src/toolbar/controls/Histogram/InteractiveIndicators.tsx

```diff
@@ -0,0 +1,61 @@
+import type { Domain } from '@h5web/shared';
+import { Drag } from '@visx/drag';
+
+import type { Size } from '../../../vis/models';
+import styles from './Histogram.module.css';
+import Indicator from './Indicator';
+
+interface Props {
+  positions: Domain;
+  size: Size;
+  onChange: (min: number, max: number) => void;
+}
+
+function InteractiveIndicators(props: Props) {
+  const { positions, size, onChange } = props;
+  const { height, width } = size;
+  const [xMin, xMax] = positions;
+
+  return (
+    <g className={styles.indicators}>
+      <Drag
+        height={height}
+        width={width}
+        x={xMin}
+        onDragEnd={({ dx }) => onChange(xMin + dx, xMax)}
+        restrict={{ xMin: 0, xMax }}
+      >
+        {({ dragStart, dragEnd, dragMove, dx }) => (
+          <g
+            onMouseMove={dragMove}
+            onMouseUp={dragEnd}
+            onMouseDown={dragStart}
+            transform={`translate(${dx}, 0)`}
```

Comment:
I followed the example of `@visx/drag`: https://airbnb.io/visx/drag-i

Reply:
For the mouse up problem, you might be able to capture/release the pointer with `onMouseDown`/`onMouseUp`. Might be worth using pointer events over mouse events, by the way (but maybe that's not supported by visx).

Reply:
Capturing the pointer fixed the `onPointerUp` not called when releasing out-of-bounds :tada: